core/src/wallet/common/database/OperationDatabaseHelper.h

```diff
@@ -41,6 +42,10 @@ namespace ledger {
         class OperationDatabaseHelper {
         public:
             static bool putOperation(soci::session& sql, const Operation& operation);
+            static bool putERC20Operation(soci::session& sql,
```

Comment:
That puts Ethereum-based code into a pretty much abstract class. Isn’t there another way?

Reply:
Oh thanks for reminding me this ! I forgot to remove it, I already moved it under `ERC20LikeAccount`